hack/verify-govet.sh

```diff
@@ -14,24 +14,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# This script is a vestigial redirection.  Please do not add "real" logic.
-
 set -o errexit
 set -o nounset
 set -o pipefail
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+
+cd "${KUBE_ROOT}"
+
+# This is required before we run govet for the results to be correct.
+# See https://github.com/golang/go/issues/16086 for details.
+make generated_files
```

Comment:
Github doesn't show it, but I literally just moved `hack/make-rules/vet.sh` to `hack/verify-govet.sh` and changed the BASH_SOURCE line to have one less `/..`